airbyte-integrations/bases/base-normalization/normalization/transform_config/transform.py

```diff
@@ -293,6 +293,7 @@ def transform_clickhouse(config: Dict[str, Any]):
         # https://docs.getdbt.com/reference/warehouse-profiles/clickhouse-profile
         dbt_config = {
             "type": "clickhouse",
+            "driver": "clickhouse-connect",
```

Comment:
This is not a valid value for 'driver' field - use 'native'. In fact, you don't need that field at all, the connector will derive the driver protocol from the port (9440).

Reply:
hrm. will try setting it explicitly - integration tests were failing because driver was being set to `None`:
```
1 check failed:
	 dbt was unable to connect to the specified database.
	 The database returned the following error:
	   >Database Error
	   Library for ClickHouse driver type None not found
```

Reply:
Ran the tests locally and ran into this error:
```
14:29:05  Compilation Error in model dedup_exchange_rate_ab1 (models/generated/airbyte_ctes/test_normalization/dedup_exchange_rate_ab1.sql)
14:29:05    No materialization 'ignore' was found for adapter clickhouse! (searched types 'default' and 'clickhouse')
```

Maybe  the approach here is be to use either view materialization, or materialize as tables and then drop the tables afterward, unless I'm misunderstanding the intent?

Reply:
Sure, let's use view materialization instead.